assets/src/stories-editor/components/block-mover/draggable.js

```diff
@@ -75,8 +89,81 @@ class Draggable extends Component {
 	 *
 	 * @param  {Object} event The non-custom DragEvent.
 	 */
-	onDragOver = ( event ) => {
+	onDragOver = ( event ) => { // eslint-disable-line complexity
+		const {
+			blockName,
+			snapLines,
+			clearSnapLines,
+			setSnapLines,
+			parentBlockElement,
+			horizontalSnaps,
+			verticalSnaps,
+		} = this.props;
+
 		const top = parseInt( this.cloneWrapper.style.top ) + event.clientY - this.cursorTop;
+		const left = parseInt( this.cloneWrapper.style.left ) + event.clientX - this.cursorLeft;
+
+		if ( top === lastY && left === lastX ) {
+			return;
+		}
+
+		const isCTABlock = 'amp/amp-story-cta' === blockName;
```

Comment:
There is a helper for this in https://github.com/ampproject/amp-wp/pull/3312

Reply:
Will rewrite once that PR is merged 👍